policyengine_us/parameters/gov/states/nm/tax/income/deductions/standard/extra_amount.yaml

```diff
@@ -0,0 +1,23 @@
+description: New Mexico provides this additional standard deduction to elderly and blind filers.
```

Comment:
```suggestion
description: New Mexico provides this additional standard deduction amount to elderly and blind filers.
```

Reply:
I don't think we need `amount` here - standard deductions are inherently amounts already. 

Please rename the files to `base.yaml` and `elderly_blind.yaml` though, or reorganize into a folder if it specifies the elderly age. In general we can pretty much mirror the federal structure and verbiage: https://github.com/PolicyEngine/policyengine-us/tree/master/policyengine_us/parameters/gov/irs/deductions/standard